pkg/check/infra_config_test.go

```diff
@@ -0,0 +1,185 @@
+package check
+
+import (
+	"testing"
+	"time"
+
+	v1 "github.com/openshift/api/config/v1"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/openshift/vsphere-problem-detector/pkg/testlib"
+	"github.com/openshift/vsphere-problem-detector/pkg/util"
+)
+
+// Desired tests:
+// - Good config (ini)
+// - Good config (yaml)
+// - Good config multi vcenter (yaml)
+// - vCenter mismatch infra vs provider (ini)
+// - vCenter mismatch infra vs provider (yaml)
+// - vCenter mismatch infra vs provider multi vcenter (ini)
+// - vCenter mismatch infra vs provider multi vcenter (yaml)
+// - FD referencing non found vCenter (ini)
+// - FD referencing non found vCenter (yaml)
+// - vCenter not used (ini)
+// - vCenter not used (yaml)
```

Comment:
Is this comment necessary? It seems most of the the tests are implemented, despite some miss ini/yaml counterpart.

Reply:
Good catch, i'll clean this up.  it was helpful for making sure i covered most cases.